File: 2st/Riyeon/글자 지우기.java
class Solution {
    public String solution(String my_string, int[] indices) {
        char[] ch = my_string.toCharArray();
        for(int i : indices){
            ch[i] = ' ';
        }
        
        String answer = "";
        for(char c : ch){
            if(c != ' ')
                answer += c;
        }
        return answer;
    }
}

File: 3st/Juram/programmer/글자 지우기.js
function solution(my_string, indices) {
    var answer = '';
    var strArr = my_string.split('');
     indices.sort((a, b) => b - a);
    for(let i=0; i<indices.length; i++){
        strArr.splice(indices[i],1);
    }
    answer = strArr.join('');
    return answer;
}